module/VuFind/src/VuFind/Sitemap/Generator.php

```diff
@@ -142,41 +163,53 @@ class Generator
      *
      * @param BackendManager $bm      Search backend manaver
      * @param SearchService  $ss      Search manager
-     * @param string         $baseUrl VuFind base URL
```

Comment:
It appears that during the course of all the refactoring, the order of these two parameters got swapped. Was that intentional? Seems like it might be easier to keep the existing order to reduce potential BC breaks, though I'm fine with the change if there's a compelling reason.

Reply:
Oops. Now refactored back. It's not perfectly back-compatible though, since the remaining new parameters are still required.

Reply:
True. I've added a note to update the changelog when I merge this... but keeping the order the same at least reduces the cognitive load of bringing things up to date. :-)